include/faabric/scheduler/FunctionCallApi.h

```diff
@@ -7,6 +7,7 @@ enum FunctionCalls
     ExecuteFunctions = 1,
     Flush = 2,
     Unregister = 3,
-    GetResources = 4
+    GetResources = 4,
+    AddPendingMigration = 5
```

Comment:
This function call is used from the master host of an application to let know other hosts running functions of the same application that a migration opportunity has been found.

Reply:
This feels like a code comment, perhaps on `sendAddPendingMigration`?